container/container.go

```diff
@@ -568,3 +568,21 @@ func (container *Container) InitDNSHostConfig() {
 		container.HostConfig.DNSOptions = make([]string, 0)
 	}
 }
+
+// UpdateMonitor updates monitor configure for running container
+func (container *Container) UpdateMonitor(hostConfig *containertypes.HostConfig) error {
```

Comment:
Not sure how I feel about this function signature.
Can we pass in just the policy?